tests/desktop/consumer_pages/test_details_page.py

```diff
@@ -14,6 +14,9 @@ class TestDetailsPage(BaseTest):
     @pytest.mark.sanity
     @pytest.mark.nondestructive
     def test_that_application_page_contains_proper_objects(self, mozwebqa):
+        if '-dev' in mozwebqa.base_url:
+            pytest.xfail("Bug 1242648 - Support info doesn't show up on an empty profile")
```

Comment:
Yeah, I am aware of that and am not crazy about it, but I wasn't sure how to make it into a conditional xfail mark because this suite still uses `pytest-mozwebqa`, so I wasn't sure if we can access the `base_url` in the `@pytest.mark`. Do you know how to do that, @davehunt?